src/main/app.js

```diff
@@ -8,34 +8,43 @@
  * @class module:brew/app~Koa
  * @alias external:Koa
  */
+const path = require("path");
 const Koa = require("koa");
+const { createReadStream } = require('fs');
+const mount = require("koa-mount");
+const serve = require("koa-static");
 const api = require("./api");
 const logger = require("koa-logger");
-const { open } = require("./db/util");
+const {open} = require("./db/util");
 
 /**
  * @function startApp
  * @param {Number} port - The port on which the app will run
  * @param {dbUrl} dbUrl - The url at which the database resides
  * @returns {undefined}
  */
-const startApp = async ({ appPort, db: { host, port, cellar } }) => {
+const startApp = async ({appPort, db: {host, port, cellar}}) => {
     /**
      * @constant {Koa} module:brew/app~app
      */
     const app = new Koa();
 
     app.use(logger());
 
-    const db = await open({ host, port, cellar });
+    // const db = await open({ host, port, cellar });
```

Comment:
Nevermind these for now - I want inject the database connection in differently somehow.